ts/Data/DataTable.ts

```diff
@@ -245,15 +245,18 @@ class DataTable implements DataEvent.Emitter {
 
     public readonly autoId: boolean;
 
-    private columns: Record<string, DataTable.Column>;
+    // @note Made this public because direct access is so handy in Highcharts
+    public columns: Record<string, DataTable.Column>;
```

Comment:
This makes modification without events possible and lacks alias support.

Reply:
Yes, I will revert this change once I see the whole picture. Will access the columns consistently through `DataTable.getColumn`.